include/zephyr/drivers/clock_control/clock_control_bflb_common.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2024 MASSDRIVER EI (massdriver.space)
```

Comment:
nit: we are now in the second halves of 2025, time flies 🤷‍♂️

Reply:
done